{{cookiecutter.project_shortname}}/tests/test_usage.py

```diff
@@ -21,11 +26,6 @@ def test_render_component(dash_app, selenium):
     # Send keys to the custom input.
     my_component.send_keys('Hello dash')
 
-    # Wait for the output callback to complete.
-    time.sleep(1)
-
-    # Get the output component.
-    output = selenium.find_element_by_id('output')
-
-    # assert the text has changed
-    assert output.text == 'You have entered Hello dash'
+    # Wait for the text to equal, if after 10 second (default) the is not
+    # equal it will fail the test.
```

Comment:
... the *text* is not equal ...
Might as well mention the `timeout` parameter by name if you're already going to the trouble of saying that 10 seconds is the default. `timeout` is pretty standard in python, but still...